source/agora/test/Ledger.d

```diff
@@ -22,13 +23,24 @@ import agora.common.Hash;
 import agora.common.Transaction;
 import agora.test.Base;
 
+/// Returns: true if all node's block heights are equal to the provided height
+private bool allHeightsEqual (API)(API[] nodes, size_t height)
```

Comment:
Suggestion: `isConsistent` and check the block headers for equality ?

Reply:
Ok.